cmeutils/plot_tools.py

```diff
@@ -0,0 +1,23 @@
+import scipy.signal as signal
+
+
+def find_peaks(data, height=None):
+    """Finds peaks in 1-D data.
+    The peaks are any value greater than the variable entered for height. The data values are 1-D values (like x, y, or z)   
+    that you wish to find the peaks of.
+    Any points/values that excede the height will be identified as peaks.
+    
+    Parameters
+    ----------
+    data : numpy.ndarray, shape (N,1)
+       Such as x, y, OR z.
+    max_height : int, default None
```

Comment:
I think the height can be a float
```suggestion
    height : float, default None
```
https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html